util/regexp-assemble/data/932220.data

```diff
@@ -0,0 +1,631 @@
+##! Please refer to the documentation at
+##! https://coreruleset.org/docs/development/regexp_assemble/.
+
+##! Word list for rule 932220 (RCE Unix command injection part 1/5)
+##!
+##! This list comes from (and should be in sync) the `unix-shell.data` file.
```

Comment:
Is it coming from that file? It does not come from 932100 + 932105?

If it actually does come from `unix-shell.data`, should we add a comment on what is the expected processing there that makes the contents of this file?